.github/workflows/int_test.yml

```diff
@@ -64,14 +65,14 @@ jobs:
       - name: Integration Test
         run: |
           touch output.log
-          go test -v -p 20 ./resources/integration_tests | tee output.log
+          go test -v -p 20 -tags=integration ./... | tee output.log
           result_code=${PIPESTATUS[0]}
```

Comment:
I think there is no need for the output anymore to detect warnings. We do that on the SDK level (if not that we should add it to the SDK level anyway)

Reply:
fixed